### PR TITLE
Editorial Review:  Update references, check wording, formatting, and such

### PR DESCRIPTION
**Pull request by Wind4Greg**
Editorial review and  edits  for  readability and clarity.  Mostly fixes  to broken  references.

***
<a href="https://pr-preview.s3.amazonaws.com/Wind4Greg/vc-di-ecdsa/pull/71.html" title="Last updated on Aug 23, 2024, 9:57 PM UTC (14375c4)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-di-ecdsa/71/d7503f5...Wind4Greg:14375c4.html" title="Last updated on Aug 23, 2024, 9:57 PM UTC (14375c4)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -920,9 +923,9 @@ <h4>Proof Verification (ecdsa-rdfc-2019)</h4>
 Let |publicKeyBytes| be the result of retrieving the
 public key bytes associated with the
 |options|.|verificationMethod| value as described in the
-Data Integrity [[VC-DATA-INTEGRITY]] specification,
-<a data-cite="vc-data-integrity#algorithms">
-Section 4: Retrieve Verification Method</a>.
+[[controller-document]] specification,
```
Comment (TallTed) [inline index.html:926]:
```suggestion
[[[controller-document]]] specification,
```

File: index.html [hunk 2]
```diff
@@ -1276,9 +1276,9 @@ <h4>Proof Verification (ecdsa-jcs-2019)</h4>
 Let |publicKeyBytes| be the result of retrieving the
 public key bytes associated with the
 |options|.|verificationMethod| value as described in the
-Data Integrity [[VC-DATA-INTEGRITY]] specification,
-<a data-cite="vc-data-integrity#algorithms">
-Section 4: Retrieving Cryptographic Material</a>.
+[[controller-document]] specification,
```
Comment (TallTed) [inline index.html:1279]:
```suggestion
[[[controller-document]]] specification,
```

File: index.html [hunk 3]
```diff
@@ -2498,8 +2498,8 @@ <h4>serializeDerivedProofValue</h4>
             <li>
 Return the <em>derived proof</em> as a string with the
 base64url-no-pad-encoding of |proofValue| as described in the
-<a href="https://www.w3.org/TR/vc-data-integrity/#multibase-0">
-Multibase section</a> of [[VC-DATA-INTEGRITY]]. That is, return a string
+<a data-cite="controller-document#multibase-0">
+Multibase section</a> of [[controller-document]]. That is, return a string
```
Comment (TallTed) [inline index.html:2502]:
```suggestion
Multibase section</a> of [[[controller-document]]]. That is, return a string
```

File: index.html [hunk 4]
```diff
@@ -2220,8 +2220,8 @@ <h4>serializeBaseProofValue</h4>
             <li>
 Initialize |baseProof| to a string with the Multibase
 base64url-no-pad-encoding of |proofValue| as described in the
-<a href="https://www.w3.org/TR/vc-data-integrity/#multibase-0">
-Multibase section</a> of [[VC-DATA-INTEGRITY]]. That is, return a string
+<a data-cite="controller-document#multibase-0">
+Multibase section</a> of [[controller-document]]. That is, return a string
```
Comment (TallTed) [inline index.html:2224]:
```suggestion
Multibase section</a> of [[[controller-document]]]. That is, return a string
```

File: index.html [hunk 5]
```diff
@@ -2512,7 +2512,7 @@ <h4>parseDerivedProofValue</h4>
 
           <p>
 The following algorithm parses the components of the derived proof value.
-The required inputs are a derived proof value (|proofValue|). A
+The required inputs are a derived proof value (|proofValue|).
```
Comment (TallTed) [inline index.html:2515]:
```suggestion
The required input is a derived proof value (|proofValue|).
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**